src/main/java/core/basesyntax/serviceimpl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService<List<String>> {
+    private final List<String> lines = new ArrayList<>();
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            reader.readLine();
```

Comment:
The first line of the file is being read and then discarded without any check. If the first line contains headers or important information, you might be losing data. If the intent is to skip a header, please add a comment to clarify this, or consider checking the contents of the line before discarding it.